wooting-macro-backend/src/lib.rs

```diff
@@ -281,13 +300,22 @@ async fn execute_macro(macros: Macro, channel: UnboundedSender<rdev::EventType>)
 /// Puts a mandatory 0-20 ms delay between each macro execution (depending on the platform).
 fn keypress_executor_sender(mut rchan_execute: UnboundedReceiver<rdev::EventType>) {
     loop {
-        plugin::util::send(&rchan_execute.blocking_recv().unwrap());
+        let received_event = match &rchan_execute.blocking_recv() {
+            Some(event) => *event,
+            None => {
+                error!("Failed to receive an event!");
+                thread::sleep(time::Duration::from_millis(200));
```

Comment:
Context needed

Reply:
hmm i almost have a feeling this looks like a merge issue when i rebased the commits

Reply:
The error I kept to log it just in case there are issues with getting events on some setups (and they show up)

Fixed [2d0f7d3](https://github.com/WootingKb/wooting-macros/pull/194/commits/2d0f7d37f7ff043122e3ecb50264cae7332c6c4a)